crates/sui-json-rpc/src/read_api.rs

```diff
@@ -760,10 +760,13 @@ impl SuiRpcModule for ReadApi {
 
 fn to_sui_transaction_events(
     fullnode_api: &ReadApi,
+    tx_digest: TransactionDigest,
     events: TransactionEvents,
 ) -> RpcResult<SuiTransactionEvents> {
     Ok(SuiTransactionEvents::try_from(
         events,
+        tx_digest,
+        None,
```

Comment:
here timestamp in `SuiTransactionEvents` is always None even after checkpoints, is it okay?

Reply:
ya for event with in the TransactionResponse, it is ok to have none for timestamp as TransactionResponse have a timestamp already. The timestamp field is mainly for Event stream